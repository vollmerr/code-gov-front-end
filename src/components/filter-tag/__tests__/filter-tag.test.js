import React from 'react';

import FilterTag from 'components/filter-tag'

import { shallow } from 'enzyme';

describe('Testing <FilterTag /> component', () => {
  const props ={
    onClick: jest.fn(),
    category: 'test-category',
    title: 'test-title',
    value: 'test-value'
  }
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<FilterTag {...props}/>);
  });
  it('renders 1 Autocomplete with to', () => {
    expect(wrapper.exists()).toEqual(true);
  });
  it('tests onClick', () => {
    wrapper.find('div.filter-tag').prop('onClick')();
    expect(props.onClick).toBeCalled();
  });
  it('should render correct', () => {
    expect(wrapper).toMatchSnapshot();
  })
})