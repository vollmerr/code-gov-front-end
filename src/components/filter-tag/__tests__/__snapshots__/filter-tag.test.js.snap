// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing <FilterTag /> component should render correct 1`] = `
<div
  className="filter-tag"
  onClick={[Function]}
>
  <div
    className="filter-tag-title"
  >
    test-title
  </div>
</div>
`;
